routes/api/contacts.js

```diff
@@ -1,25 +1,137 @@
-const express = require('express')
+const express = require("express");
+const { v4: uuidv4 } = require("uuid");
 
-const router = express.Router()
+const {
+  listContacts,
+  getContactById,
+  addContact,
+  removeContact,
+  updateContact,
+} = require("../../models/contacts");
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const validateContact = require("../../utils/validator");
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const router = express.Router();
 
-router.post('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+// GET contacts list =================================
+router.get("/contacts", async (_, res) => {
```

Comment:
```suggestion
router.get("/", async (_, res) => {
```
Ta zmiana zakłada że wzmieniłaś w głównym pliku tak jak sugerowałem

Reply:
Czy rozumiesz zmianę którą sugeruję? Jak nie to dopytaj tutaj i daj znać na slaku że mam komentarz do przeczytania.

Reply:
Zmiana oczywista.
Poprawione.